docker/CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
 # Changelog
 All notable changes to the docker containers will be documented in this file.
 
+### 2025-11-21
+- Updated `Alert_on_HTTP_Response_Code_Errors.js` to pass the extracted status code to setEvidence.
```

Comment:
`… to work with GraalVM JavaScript engine.` would be more clear IMO what the fix is.

Reply:
Updated.